clusterloader2/pkg/measurement/common/profile.go

```diff
@@ -159,33 +160,41 @@ func (p *profileMeasurement) String() string {
 	return p.name
 }
 
-func (p *profileMeasurement) gatherProfile(c clientset.Interface) (measurement.Summary, error) {
+func (p *profileMeasurement) gatherProfile(c clientset.Interface) ([]measurement.Summary, error) {
 	profilePrefix := fmt.Sprintf("%s_%s", p.config.componentName, p.name)
```

Comment:
Can we add host to the profilePrefix? Otherwise it would be impossible to tell from which host (master) the profile comes.

Reply:
Done!
thank you for your comment.